.github/workflows/test_onnx_ops.yml

```diff
@@ -53,25 +54,31 @@ jobs:
 
       # Run tests and output new config files as needed.
       - name: Run ONNX ops test suite
+        continue-on-error: true
```

Comment:
This workflow will rarely "fail" now. We might want some way to treat _test_ failures as okay, but still report _test runner_ failures (e.g. out of disk space, failed to download files, etc.).